src/app/components/modules/UserWallet.jsx

```diff
@@ -83,6 +83,38 @@ class UserWallet extends React.Component {
         this.props.claimRewards(account);
     };
 
+    getCurrentApr = gprops => {
+        // The inflation was set to 9.5% at block 7m
+        const initialInflationRate = 9.5;
+        const initialBlock = 7000000;
+
+        // It decreases by 0.01% every 250k blocks
+        const decreaseRate = 250000;
+        const decreasePercentPerIncrement = 0.01;
+
+        // How many increments have happened since block 7m?
+        const headBlock = gprops.head_block_number;
+        const deltaBlocks = headBlock - initialBlock;
+        const decreaseIncrements = deltaBlocks / decreaseRate;
+
+        // Current inflation rate
+        let currentInflationRate =
+            initialInflationRate -
+            decreaseIncrements * decreasePercentPerIncrement;
+
+        // Cannot go lower than 0.95%
+        if (currentInflationRate < 0.95) {
+            currentInflationRate = 0.95;
+        }
+
+        // Now lets calculate the "APR"
+        const virtualSupply = gprops.virtual_supply.split(' ').shift();
+        const totalVestingFunds = gprops.total_vesting_fund_steem
+            .split(' ')
+            .shift();
+        return virtualSupply * currentInflationRate * 0.15 / totalVestingFunds;
```

Comment:
The value of 15% can be obtained from the dynamic global property object, `vesting_reward_percent`. It should be read from there so that if the value changes in a future release, this code will remain accurate.

Reply:
@mvandeberg  This is what I get from the variable gprops, can't see vesting_reward_percent
`{
	"virtual_supply": "359267635.665 STEEM",
	"reverse_auction_seconds": 900,
	"confidential_supply": "0.000 STEEM",
	"num_pow_witnesses": 172,
	"total_reward_fund_steem": "0.000 STEEM",
	"init_sbd_supply": "0.000 SBD",
	"total_reward_shares2": "0",
	"pending_rewarded_vesting_shares": "752017824.390962 VESTS",
	"sbd_interest_rate": 0,
	"head_block_id": "022324c64e2797730936c63faa99215f4c273014",
	"time": "2019-08-25T09:25:48",
	"recent_slots_filled": "340282366920938463463374607431768211455",
	"confidential_sbd_supply": "0.000 SBD",
	"participation_count": 128,
	"sbd_stop_percent": 1000,
	"vote_power_reserve_rate": 10,
	"maximum_block_size": 65536,
	"current_witness": "cervantes",
	"current_supply": "323341049.277 STEEM",
	"pending_rewarded_vesting_steem": "373711.639 STEEM",
	"delegation_return_period": 432000,
	"current_aslot": 35963316,
	"total_vesting_fund_steem": "200`

Reply:
As the missing fields were added to the API, I've updated the code to use the value from the dynamic global properties